Source/gamemenu.cpp

```diff
@@ -31,39 +31,35 @@ void GamemenuSpeed(bool bActivate);
 /** Contains the game menu items of the single player menu. */
 TMenuItem sgSingleMenu[] = {
 	// clang-format off
-    // dwFlags,      pszStr,              fnMenu
-	{ GMENU_ENABLED, N_("Save Game"),     &gamemenu_save_game  },
-	{ GMENU_ENABLED, N_("Options"),       &GamemenuOptions    },
-	{ GMENU_ENABLED, N_("New Game"),      &GamemenuNewGame   },
-	{ GMENU_ENABLED, N_("Load Game"),     &gamemenu_load_game  },
-#ifndef NOEXIT
-	{ GMENU_ENABLED, N_("Quit Game"),     &gamemenu_quit_game  },
-#endif
-	{ GMENU_ENABLED, nullptr,              nullptr             }
+    // dwFlags,      pszStr,          fnMenu

```

Comment:
if you decided to align this stuff, please align the header too ;)

Reply:
I actually did, but didn't spot that someone ad used spaces instead of tab for the indentation.